src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2780,6 +2780,8 @@ bool SvgLoader::header()
     loaderData.svgParse = (SvgParser*)malloc(sizeof(SvgParser));
     if (!loaderData.svgParse) return false;
 
+    loaderData.svgParse->flags = (SvgStopStyleFlags)0x0;
```

Comment:
How about declare it?

enum class SvgStopStyleFlags
{
    StopDefault = 0x0,
    StopOpacity = 0x01,
    StopColor = 0x02
};

Reply:
i updated patch using default flag